src/dataLoaders/components/verifyStep/ConnectionInformation.tsx

```diff
@@ -56,15 +57,25 @@ class ConnectionInformation extends PureComponent<Props> {
     }
   }
 
-  private get additionalText(): string {
+  private get additionalText(): any {
```

Comment:
This is fine, although the use of `any` seems lazy here but it's not a blocker.

Reply:
I think the problem is if we define it, the one i added is not a string, and the others are strings. Idk if we could do string || object? i think any just makes it easier for now